components/home/Tools.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ScaffoldContainer from '../ScaffoldContainer';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    '& .MuiCardContent-root': {
+      padding: '0.5rem 1rem 2rem',
+    },
+    '& .MuiGrid-spacing-xs-10': {
+      padding: '1rem',
+    },
+    paddingLeft: theme.spacing(5),
+    paddingRight: theme.spacing(5),
+  },
+  heading: {
```

Comment:
It looks like you may be able to use some of the predefined theme typography (found in `theme.js`) instead of creating the styles -- I'll comment an example below. Also, material-ui gives us access to theme values and functions in the styles, so instead of using explicit  '1rem' you could also do `theme.spacing(1)` which will use a spacing fn. [More about that here](https://material-ui.com/customization/spacing/)